src/player.ts

```diff
@@ -405,4 +405,41 @@ export class Player extends NativeInstance<PlayerConfig> {
   castStop = () => {
     PlayerModule.castStop(this.nativeId);
   };
+
+  /**
+   * Sets the playback speed of the player. Fast forward, slow motion and reverse playback are supported.
+   * @note
+   * - Slow motion is used by values between `0` and `1`.
+   * - Fast forward by values greater than `2`.
+   * - Slow reverse is used by values between `0` and `-1`, and fast reverse is used by values less than `-1`. iOS and tvOS only.
+   * @note
+   * Negative values are ignored during Casting.
+   * @note
+   * During reverse playback the playback will continue until the beginning of the active source is
+   * reached. When reaching the beginning of the source, playback will be paused and the playback
+   * speed will be reset to its default value of `1`. No {@link PlaybackFinishedEvent} will be
+   * emitted in this case.
+   *
+   * @param playbackSpeed - The playback speed level to set.
+   */
+  setPlaybackSpeed = (playbackSpeed = 1) => {
+    PlayerModule.setPlaybackSpeed(this.nativeId, playbackSpeed);
+  };
+
+  /**
+   * @returns The player's current playback speed.
+   */
+  getPlaybackSpeed = async (): Promise<number> => {
+    return PlayerModule.getPlaybackSpeed(this.nativeId);
+  };
+
+  /**
+   * Checks the possibility to play the media at specified playback speed.
+   * @param playbackSpeed The playback speed to check.
+   * @returns `true` if it's possible to play the media at the specified playback speed, otherwise `false`.
+   * @platform iOS, tvOS
+   */
+  canPlayAtPlaybackSpeed = async (playbackSpeed: number): Promise<boolean> => {
+    return PlayerModule.canPlayAtPlaybackSpeed(this.nativeId, playbackSpeed);
+  };
```

Comment:
This is not implemented for Android, so it causes a runtime error.

Please add an early return when calling this on Android.

The value of the returns should be TBD. I picked `undefined` as we don't know if it is `true` or `false`.

@strangesource what do you think about this?

```suggestion
  canPlayAtPlaybackSpeed = async (
    playbackSpeed: number
  ): Promise<boolean | undefined> => {
    if (Platform.OS === 'android') {
      console.warn(
        `[Player ${this.nativeId}] Method canPlayAtPlaybackSpeed is not available for Android. Only iOS and tvOS devices.`
      );
      return undefined;
    }
    return PlayerModule.canPlayAtPlaybackSpeed(this.nativeId, playbackSpeed);
  };
```

Reply:
Undefined makes sense IMO. 👍 Let's make sure we have it as part of the documentation.

Reply:
Address in cb36609afc66508d809b0f9c1aee30054a49b43e  
Update documentation in 0c7129df518386d0c55dc614f1986a022b3ecf83